src/nyc_trees/js/src/map.js

```diff
@@ -75,7 +80,7 @@ function create(options) {
         zoomControl.removeFrom(map);
     }
 
-    return map;
+    return {map: map, zoomControl: zoomControl};
```

Comment:
Rather than returning and removing `zoomControl` I suggest you return and remove `$controlsContainer`, so we remove whatever map controls are there. There's an issue open to add search to this page, and the zoom control might be removed for mobile.